pyproject.toml

```diff
@@ -69,6 +69,36 @@ mkdocs-material = "^5.5.5"
 # HACK https://readthedocs.org/api/v2/build/11653917.txt
 # TODO Remove this dependency when mkdocstrings supports pymdown-extensions==8.0
 pymdown-extensions = "<8.0,>=6.3"
+tox = "^3.19.0"
```

Comment:
Remove tox (explained below), but add poethepoet please.

Also, add it above of the `HACK` line (around line 64)

Reply:
Done.